### ISSUE TITLE
Bug: LV-N Nerv shroud only half deployed on pre-existing spacecraft, no option to deploy.

### ISSUE THREAD

**Issue by paperbw** [PorktoberRevolution/ReStocked#575]
Has anyone else had this problem?

![20190401103458_1](https://user-images.githubusercontent.com/49140619/55340405-3b51e480-546a-11e9-84fa-cb7044ccb7fd.jpg)

**Comment by drewcassidy** [comment]
Yes this is known. Fixing it is finicky, because it’s difficult to reproduce. Was this just a vessel made in the stock game before installing restock?

**Comment by paperbw** [comment]
Yeah stock vessels made prior to installing restock.

**Comment by xot1643** [comment]
Had the same issue with Restock for existing crafts. Had also the same issue with RealPlume.
It can be fixed with editing the persistent or quicksave file to remove the remaining shroud.
It is a bit complicated if you have never edited or simply looked at it but it is doable.

**Comment by paperbw** [comment]
Yeah I could definitely go in and manually remove the shrouds from the persistent. Does that justify closing the thread or should I keep it open?

**Comment by drewcassidy** [comment]
No this should be left open. We're looking into some solutions for this to work around the problems with the base game

**Comment by joshbilderback** [comment]
My issue is similar.  Craft without shrouds suddenly have them.  NERV had half shrouds.
I should point out this happened on craft where I had Restock on 1.7.3, and then removed it for 1.8.  May not be super helpful but I was told to post my findings.

My fix for editing the save file was changing the **activejettisonName** attribute for the affected engines
.
Rhinos had 'Fairing375', but game expects it to be 'obj_fairing'
SSMEs had 'Fairing175', but game expects it to be 'Fairing'
Terriers with normal attach point expected 'ShortShroud'
No issues with Wolfhounds.
NERV engines having 2 entries had 'fairingL' missing, or blank.

**Comment by ChrisAdderley** [comment]
So we resolved those changes with version 0.1.1 or around there. Those names are definitely not in the current release version, aside from the last:
> NERV engines having 2 entries had 'fairingL' missing, or blank.
Yeah this seems like it's a bug with Squad's implementation of the autoshroud. Sometimes it blanks out the transform name for no apparent reason when there are two instances of the module on the part. It's quite vexing and is why this issue is still open.
